config/routes.rb

```diff
@@ -2,4 +2,11 @@
   resources :users, only: [:new, :create, :show, :edit, :update, :destroy]
   resources :budgets, only: [:new]
   root to: "home#show"
+  get "/login", to: "sessions#new"
+  post "/login", to: "sessions#create"
+  resources :sessions, only: [:destroy]
+  scope "/:locale", locale: /en/ do
+    resources :users, only: [:new]
+    resources :budgets, only: [:new]
+  end
```

Comment:
Are lines 7-11 needed?
Also, they are not in scope for this PR

Reply:
Fixed with sha: 9c21eb7